System.Compression.Primitives/System/IO/Compression/Brotli/BrotliNative.cs

```diff
@@ -0,0 +1,308 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+namespace System.IO.Compression
+{
+    /// <summary>
+    /// This class provides declaration for constants and PInvokes as well as some basic tools for exposing the
+    /// native Brotli library to managed code.
+    /// </summary>
+   internal static partial class BrotliNative
+    {
+        public enum BrotliDecoderResult : int
```

Comment:
Nit: Specifying int is redundant.

"The default underlying type of enumeration elements is int."